safenode/src/node/api.rs

```diff
@@ -252,6 +252,9 @@ impl Node {
                 self.send_response(Response::Cmd(resp), response_channel)
                     .await;
             }
+            Cmd::Replicate(_) => {
+                warn!("Replication cmd shall not be received here");
```

Comment:
Does the msg shall actually be that the "Cmd is not processed here", "ignored"? because the Cmd effectively will be always received there as your are monitoring all the Cmds. Plus, because of the same reason, this shall not be a `warn`, at most it can be a `debug`, or just `trace`, there is nothing to warn about here as far as I can see?

Reply:
`Cmd::Replicate` got handled within SwarmDriver already, which won't be sent to `Node` level.
Hence put it as a warning as it may indicates a leak of handling.

Reply:
yeah, I saw that, that is a huge problem, changing the flow of where things are handled, you'll need to handle the Cmd here since this is the place where Cmds are handled, otherwise you are breaking the design of the flow completely.